vagrant/provision.sh

```diff
@@ -31,7 +31,7 @@ chmod 777 /var/maestro
 echo "#!/bin/bash
 . /etc/profile.d/envvars.sh
 cd $MAESTRO_SRC
-$GOBIN/maestro
+exec $GOBIN/maestro
```

Comment:
```suggestion
exec ${GOBIN}/maestro
```

Reply:
What does the `{` and `}` do in this case?

Reply:
It's known as brace expansion. I got it and unsert variables confused, but it's imo it's good bash programming practive to use them: https://stackoverflow.com/questions/8748831/when-do-we-need-curly-braces-around-shell-variables/8748880#8748880

Reply:
Unnecessary in this case however.